app/src/main/kotlin/compiler/common/Constant.kt

```diff
@@ -0,0 +1,17 @@
+package compiler.common
+
+interface Constant {
```

Comment:
I think we could instead have `UnspecifiedConstant` as `IFTNode` but this also looks reasonable.

Reply:
But then we'd probably need to have `Add(Const(variables offset), UnspConst(spilled offset))` in `prologue` and `epilogue` and instead we only have `Const` which is more efficient. So that's perfectly fine.